src/SkillsForm.jsx

```diff
@@ -16,23 +15,18 @@ class SkillForm extends React.Component {
     super(props);
     this.state = {
       skill_name: "",
-      skill_type: ""
+      skill_type: skillTypes[0].value
     };
-    this.logChange = this.logChange.bind(this);
     this.onChange = this.onChange.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
   }
 
-  logChange(val) {
-    console.log("Selected:" + val);
-  }
-
   onChange(key, value) {
     this.setState({ [key]: value});
   }
 
-  onSubmit(ev) {
-    ev.preventDefault();
+  onSubmit(env) {
```

Comment:
I might be confused, but I thought `ev` stood for `event`? Why change this to `env`? That makes me think "environment".